_rules/inline-link-in-paragraph-has-distinguishable-style-on-hover-and-focus-548868.md

```diff
@@ -0,0 +1,154 @@
+- a difference between the [foreground colors][foreground color] that has at least a 3:1 [contrast ratio](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio); and
+- a [distinguishing style][] both when the target element [gains focus][focused] and the target element is [hovered][].
+
+## Assumptions
+
+- The link is distinguishable from the rest of the text with color, which means it fails SC 1.4.1 when there is not another way to distinguish it.
+- The 3:1 contrast difference between text is minimal to what would be sufficient to meet WCAG 2.0. This value is part of [technique G183](https://www.w3.org/WAI/WCAG21/Techniques/general/G183), but is not specified in the [1.4.1 success criterion](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html).
+- Any change in font is sufficiently distinguishable, and fonts are loaded when they are present.
+- If multiple colors are used in the visible text nodes of the block of content then color can not be a distinguishing factor.
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+- [Understanding Success Criterion 1.4.1: Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)
+- [G183: Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them](https://www.w3.org/WAI/WCAG21/Techniques/general/G183)
+- [F73: Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision](https://www.w3.org/WAI/WCAG21/Techniques/failures/F73)
+- [C15: Using CSS to change the presentation of a user interface component when it receives focus](https://www.w3.org/WAI/WCAG21/Techniques/css/C15)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This link, that is a descendant of a paragraph element, has a text contrast of more than 3:1 compared to the other text in the paragraph. When the link receives focus, an underline appears. When the link receives hover, an underline appears.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		text-decoration: none;
+		color: #d14826;
+		background-color: white;
+	}
+	a:hover,
+	a:focus {
+		text-decoration: underline;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This link, that is a descendant of a paragraph element, has no visual cues of being recognized as a link with the underline removed.
+
+```html
+<style>
+	a.test {
+		text-decoration: none;
+	}
+</style>
+<p>Read about WAI on the <a class="test" href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
```

Comment:
I am missing a failed example where the 2 foreground colours don't have a contrast of at least 3:1

Reply:
Added such an examepl